src/test/java/com/spotify/futures/CompletableFuturesTest.java

```diff
@@ -429,4 +431,23 @@ public void combine5_exceptional() throws Exception {
     exception.expectCause(isA(IllegalStateException.class));
     getCompleted(future);
   }
+
+  private static <T> Matcher<CompletionStage<T>> completesTo(final T expected) {
+    return completesTo(is(expected));
+  }
+
+  private static <T> Matcher<CompletionStage<T>> completesTo(final Matcher<T> expected) {
+    return new CustomTypeSafeMatcher<CompletionStage<T>>("complete to" + String.valueOf(expected)) {
+      @Override
+      protected boolean matchesSafely(CompletionStage<T> item) {
+        try {
+          final T value = item.toCompletableFuture().get(1, TimeUnit.SECONDS);
```

Comment:
You could also depend on `com.jayway.awaitility:awaitility:1.7.0` which provides matchers like this, makes completion time configurable and provides reasons (timeout vs exceptionally completed etc) when it fails.